pkg/types/exchange.go

```diff
@@ -65,6 +65,8 @@ type Exchange interface {
 	ExchangeMarketDataService
 
 	ExchangeTradingService
+
+	ExchangeTransferService
```

Comment:
I want to make the ExchangeTransferService interface as an optional interface, that way we can add new exchange faster and easier, what do you think?

check the `transfer-history` command to see that implementation checking?

Reply:
I think you should ask everyone implement all interfaces because bbgo is a really general framework. When you publish a new release and say `bbgo support the new exchange`. Everyone thinks all functions are supported, and they just have to create a new config. Most people may not read all your release notes. They just want you finish the whole job.